src/middle/Program.ml

```diff
@@ -31,20 +31,16 @@ type ('a, 'b) t =
   ; input_vars: (string * 'a SizedType.t) list
   ; prepare_data: 'b list (* data & transformed data decls and statements *)
   ; log_prob: 'b list (*assumes data & params are in scope and ready*)
-  ; generate_quantities: 'b list (* assumes data & params ready & in scope*)
+  ; generate_quantities: 'b list
+        (* assumes data & params ready & in scope*)
+        (* TODO the following two items are really backend-specific *)
```

Comment:
Agree these should be moved to something on the backend, but this might be better just as an issue somewhere or as a top level comment